website/ibm.erb

```diff
@@ -86,8 +86,8 @@
             <li<%= sidebar_current("docs-ibm-datasource-cis-domain") %>>
               <a href="/docs/providers/ibm/d/cis_domain.html">cis_domain</a>
             </li>
-            <li<%= sidebar_current("docs-ibm-datasource-cis-firewall") %>>
-              <a href="/docs/providers/ibm/d/cis_firewall.html">cis_firewall</a>
```

Comment:
data source is `ibm_cis_firewall` it should be inline with datasource name.. Please revert it

Reply:
changed